modules/ref-dashboard-configuration-options.adoc

```diff
@@ -11,41 +11,57 @@ The {productname-short} dashboard includes a set of core features enabled by def
 |===
 | Feature | Default | Description
 | `dashboardConfig:
-enablement` | `true` | Enables admin users to add applications to the {productname-short} dashboard *Application → Enabled* page. To disable this ability, set the value to `false`.
+disableAcceleratorProfiles` | `false`| Shows the *Settings → Accelerator profiles* option in the dashboard navigation menu. To hide this menu option, set the value to `true`.
 | `dashboardConfig:
-disableInfo` | `false` | On the *Applications → Explore* page, when a user clicks on an application tile, an information panel opens with more details about the application. To disable the information panel for all applications on the *Applications → Explore* page , set the value to `true`.
+disableBiasMetrics` | `false` | Shows the *Model Bias* tab on the *Model Serving* page. To hide this tab, set the value to `true`. 
```

Comment:
This PR may or may not go in for 2.16. not sure if that impacts this pr or not. but it will change the name of the feature flag

https://issues.redhat.com/browse/RHOAIENG-14126

Reply:
Thanks for the "heads up" @Gkrumbach07 . I'm watching that ticket now and will update the doc when it is implemented.